scripts/pre_push_hook.py

```diff
@@ -122,10 +146,10 @@ def _collect_files_being_pushed(ref_list, remote):
         ref_list: list of references to parse (provided by git in stdin)
         remote: the remote being pushed to
     Returns:
```

Comment:
Yes.  I guess this is what most of us do... however I also thought that people stay on the branch that they want to push (as I tend to do that)  and I was wrong there :wink:

I'll write a comment  in there